Source/Meadow.Foundation.Core/Communications/ByteCommsSensorBase.cs

```diff
@@ -12,7 +12,7 @@ public abstract class ByteCommsSensorBase<UNIT> :
         PollingSensorBase<UNIT>, IDisposable where UNIT : struct
     {
         /// <summary>
-        /// Peripheral object, i.e. an I2CPeripheral or SpiPeripheral
+        /// Bus communications object, i.e. an I2cCommunications or SpiCommunications
         /// </summary>
         protected IByteCommunications? Peripheral { get; set; }
```

Comment:
Should this still be called Peripheral?

Reply:
great call - I'm on it - I'm thinking `ByteComms`